jax/_src/path.py

```diff
@@ -19,8 +19,10 @@
 
 try:
   import etils.epath as epath
+  epath_installed = True
```

Comment:
I guess you could use `epath is not None` instead of `epath_installed`?

Reply:
I think having a `epath_installed` is clearer, because it is imported in another file and used multiple times.